sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -82,6 +83,16 @@ class CmisApi(XcvrApi):
     LowPwrRequestSW = 4
     LowPwrAllowRequestHW = 6
 
+    # Default caching disabled; control via classmethod
+    cache_enabled = False
```

Comment:
still need this?

Reply:
If we enable cache for all vendors, this is not needed, will confirm with @prgeor